Python/Product/PythonTools/PythonTools/Repl/BasePythonReplEvaluator.cs

```diff
@@ -1783,6 +1785,19 @@ class MemberResults {
         }
 
 #if DEV14_OR_LATER
+        internal static Version VSInteractiveVersion {
+            get {
+                if (_vsInteractiveVersion == null) {
+                    _vsInteractiveVersion = typeof(IInteractiveWindow).Assembly
+                        .GetCustomAttributes(typeof(AssemblyInformationalVersionAttribute), false)
+                        .OfType<AssemblyInformationalVersionAttribute>()
+                        .Select(a => { Version v; return Version.TryParse(a.InformationalVersion, out v) ? v : null; })
+                        .FirstOrDefault(v => v != null) ?? new Version();

```

Comment:
Incidentally, PTVS 3.0 won't have this hack. We'll just assume that you're running at least VS 2015 U1.